semgrep_output_v1.atd

```diff
@@ -804,6 +811,7 @@ type finding <ocaml attr="deriving show"> = {
   syntactic_id: string;
   (* since semgrep 0.98 *)
   ?match_based_id: string option;
+  ?hashes: finding_hashes option;
```

Comment:
im not sure what the use case for this information is, but you may want to add a "since semgrep ..." as above, which will help document how our output interface has evolved

Reply:
left my best-guess at when this will get introduced, but will try to remember to adjust it once it's actually released in semgrep